EIPS/eip-5173.md

```diff
@@ -0,0 +1,429 @@
+
+![Figure 3: Losing owners](../assets/eip-5173/Losing_owners.jpeg)
+
+*Figure 3: Losing owners*
+
+### Single vs Multigenerations
+
+In a single generation reward, the new buyer/owner receives a share of the next single generation's realized profit only. In a multigenerational reward system, buyers will have future rewards years after their purchase. The NFT should have a long-term growth potential and a substantial dividend payout would be possible in this case. 
+
+We propose that the marketplace operator can choose between a single generational distribution system and a multigenerational distribution system.
+
+### Direct FR Payout by the Seller vs Smart Contract-managed Payout
+
+FR payouts directly derived from the sale proceeds are immediate and final. As part of the fraud detection detailed later in the Security Considerations section, we selected a method in which the smart contract calculates all the FR amounts for each generation of previous owners, and handles payout according to other criteria set by the marketplace, such as reduced or delayed payments for wallet addresses with low scores, or a series of consecutive orders detected using a time-heuristic analysis. 
+
+### Equal vs Linear Reward Distributions
+#### Equal FR Payout
+
+![Figure 4: Equal, linear reward distribution](../assets/eip-5173/Total_FR_Payout_Distribution-flat.png?raw=true)
+
+*Figure 4: Equal, linear reward distribution*
+
+FR distributions from the realization of profits by later owners are distributed equally to all eligible owners (*Figure 4*). The exponential reward curve, however, may be more desirable, as it gives a slightly larger share to the newest buyer. Additionally, this distribution gives the earliest generations the largest portions as their FR distributions near the end, so they receive higher rewards for their early involvement, but the distribution is not nearly as extreme as one based on arithmetic sequences (*Figure 5*). 
+
+This system does not discriminate against any buyer because each buyer will go through the same distribution curve.
+
+#### Straight line arithmetic sequence FR payout
+
+![Figure 5: Arithmetic sequence distribution](../assets/eip-5173/Arithmetic_Sequence_FR_Payout_Distribution.png?raw=true)
+
+*Figure 5: Arithmetic sequence distribution*
+
+The profit is distributed according to the arithmetic sequence, which is 1, 2, 3, ... and so on. The first owner will receive 1 portion, the second owner will receive 2 portions, the third owner will receive 3 portions, etc. 
+
+## Backwards Compatibility
+
+This proposal is fully compatible with current [EIP-721](./eip-721.md) standards and [EIP-2981](./eip-2981.md). It can also be easily adapted to work with [EIP-1155](./eip-1155.md).
+
+## Test Cases
+
+[This folder](../assets/eip-5173/Implementation) contains the reference implementation for EIP-5173.
```

Comment:
Unfortunately our markdown renderer can't handle links to entire folders, so you'll have to pick a single file to link to.